### DISCUSSION TITLE
Colab Notebook

### DISCUSSION THREAD

**Discussion by omnia487** [securefederatedai/openfederatedlearning#397] [Q&A]
can anyone mention the colab notebook link of any one of [tf_2dunet] or [tf_3dunet_brats] templates?

**Comment by alexey-gruzdev** [comment]
@omnia487 if we talk about Google Colab examples, we don't have them for [tf_2dunet] or [tf_3dunet_brats] templates.

**Reply by omnia487** [reply]
Is there any colab example using the BraTs dataset?

**Reply by alexey-gruzdev** [reply]
@omnia487 right now, only with Linear Regression & synthetic data. No example on BRATS data, since the dataset is not public, and we cannot distribute it as a part of Colab/something else.

**Reply by omnia487** [reply]
i request the data and I get it already from the portal but i have a problem with the tf_2dunet template related to the dataset 
![156519762-74d24fc0-752d-499b-b402-31c2e318bcd2](https://user-images.githubusercontent.com/89255896/160114476-717a7690-e11c-4a67-9593-b5823b9449ca.png)

**Reply by omnia487** [reply]
although there isn't any problem with the other templates

**Reply by alexey-gruzdev** [reply]
Since the topic of this thread is Colab Notebook, let's discuss it in dedicated issue, and please mark this as resolved if it actually is.

**Reply by omnia487** [reply]
I already mentioned this problem in many dedicated issues but  there is no answer
